src/Vipps.net/Infrastructure/IVippsConfigurationProvider.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Vipps.net.Infrastructure
+{
+    public interface IVippsConfigurationProvider
```

Comment:
Any reason why we can't just use the `VippsConfigurationOptions` directly? What does the Provider abstraction provide?

Reply:
Removed the abstraction, are now using the VippsConfigurationOptions directly